libs/coin-modules/README.md

```diff
@@ -0,0 +1,41 @@
+Coin Modules
+============
+
+Alpaca compatibility
+--------------------
+
+### Architectural Decision
+The following concepts are mandatory for a CoinModule to be *Alpaca compliant*.
+#### A CoinModule has the following directories/modules
+* `api`: interface for exposing the CoinModule to a backend service (a.k.a. Alpaca)
```

Comment:
Not a blocker, just I think `api` was mostly used for what you're describing in network today (at least EVM for sure)